portfolio/src/main/webapp/index.html

```diff
@@ -149,32 +142,21 @@ <h3> Why Computer Science? </h3>
               around us. Fortunately, since technology continues to rapidly evovle before our very eyes,
               there is always so many new things to learn. In order to continue to build upon my current coding 
               knowledge,this summer, I have the opportunity to be apart of Google's STEP, Student Training in 
-              Engineering Program, internship. 
-          </p>
+              Engineering Program, internship. </p>
 
           <div>  </div>
 
           <h3> Learn More About Me</h3>
```

Comment:
Seems like this doesn't fully show: https://screenshot.googleplex.com/ApEzYz9tDfY